packages/portal-data/lib/transform.js

```diff
@@ -57,12 +57,15 @@ exports.fromService = function ({ service, instances, packages }) {
     currentMetrics: [],
     connections: service.service_dependency_ids,
     package: packages ? exports.fromPackage(packages) : {},
-    parent: service.parent_id || ''
+    parent: service.parent_id || '',
+    active: service.active
   };
 };
 
 exports.toService = function (clientService) {
-  return {
+  // wat??
+  return JSON.parse(JSON.stringify({
```

Comment:
I have the same question. rethinkdb throws when some fields are `undefined`. I don't know why :/